src/test/java/com/mentoring/LoginTest.java

```diff
@@ -63,11 +68,13 @@ public void wrongUserName() {
     @Test(groups = "smoke")
     public void wrongPassword() {
         String validUserName = User.generateRandomUsername();
+
         registrationAndLogout(validUserName, VALID_PASSWORD)
                 .fillLoginForm(validUserName, WRONG_PASSWORD)
                 .clickLoginButton();
+
         NotificationContainer notificationContainer = new PageFactory().getNotificationContainer(driver);
-        softAssertion.assertThat(notificationContainer.isAButtonLabelEqualsTo(Messages.BAD_CREDENTIALS))
+        softAssertion.assertThat(CommonAssertions.hasButtonsListErrorText(notificationContainer, Messages.BAD_CREDENTIALS))
```

Comment:
Not quite what I meant. I'd move the assertion as well. It would look something like this:
CommonAssertions.assertNotificationMessageIsCorrect(softAssertion, Messages.BAD_CREDENTIALS);
So it could be a totally general notification assertion method.